motioneye/sendtelegram.py

```diff
@@ -15,27 +15,19 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import datetime
-import io
 import logging
 import os
 import re
 import signal
 import socket
-import sys
 import time
 
 import pycurl
 from tornado.ioloop import IOLoop
 
-from motioneye import config, mediafiles, motionctl, settings, utils
+from motioneye import config, mediafiles, motionctl, settings, utils, meyectl
```

Comment:
Alphabetical sorting:
```suggestion
from motioneye import config, mediafiles, meyectl, motionctl, settings, utils
```